apps/strapi-cms/src/components/components/pre-header-button.json

```diff
@@ -22,7 +22,6 @@
     "variant": {
```

Comment:
We may as well remove variant for PreHeaderButton since it's just ignored and always set to 'text'